app/assets/stylesheets/mixins/_white-links.scss

```diff
@@ -0,0 +1,30 @@
+// TODO (@bfirsh): refactor frontend_toolkit external link mixin to work with
```

Comment:
White external links aren't needed for publications, they were added for consultations. Removing this and the images from this PR.